### PR TITLE
feature/23

### PR DESCRIPTION
**Pull request by EduardStreltsov**
- added PlaceController
- added place_form.html

### CODE REVIEW

File: src/main/java/co/gojohnny/web/controller/PlaceController.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/places")
+public class PlaceController {
+
+    @Autowired
```
Comment (boy4uck) [inline src/main/java/co/gojohnny/web/controller/PlaceController.java]: @EduardStreltsov, please use constructor-based DI

**Comment by boy4uck** [comment]
@EduardStreltsov, you have to resolve conflicts first

File: build.gradle [hunk 2]
```diff
@@ -29,4 +29,8 @@ dependencies {
     //helpers
     compile 'org.projectlombok:lombok:1.16.12'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.8.8.1'
+
+    //tomcat bugfixes
+    compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.25'
+    testCompile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.25'
```
[the remark this answers is not in the document]
Comment (EduardStreltsov) [reply build.gradle]: because my tomcat doesn't start without these deps. I forgot to remove them.

**Comment by boy4uck** [comment]
@EduardStreltsov, it seems that you did not do a correct `rebase`. Please review the files changes. It contains a lot of unrelated changes. Again, try to do `rebase` on `master` branch and then use `force push`

**Comment by EduardStreltsov** [comment]
Is it correct now?

**Pull request merged**